.travis.yml

```diff
@@ -32,5 +32,19 @@ branches:
 
 jobs:
     include:
-        - python: 3.6
-          script: make test
+        - stage: default
+          name: numpy
+          python: 3.6
+          script:
+              - make test
+        - stage: backend
```

Comment:
Can we restrict to a single stage to keep testing in parallel? Otherwise test results will have higher latency.